File: TeachersGuardAPI/Domain/Entities/Schedule.cs
namespace TeachersGuardAPI.Domain.Entities
{
    public class Schedule
    {
        public required string UserId { get; set; }
        public required string PlaceId { get; set; }
        public required string ScheduleId { get; set; }
        public required string Start {  get; set; }
        public required string End { get; set; }
        public required List<DayOfWeek> DayOfWeek { get; set; }
    }
}

File: lib/domain/entities/schedule.dart
import 'package:teachersguard/domain/entities/entities.dart';

class Schedule {
  final String id;
  final Place place;
  final String begin;
  final String end;
  final List<int> dayOfWeek;

  Schedule({
    required this.id,
    required this.place,
    required this.begin,
    required this.end,
    required this.dayOfWeek});
}
